javascript/javascript1/week1/age-ify.js

```diff
@@ -0,0 +1,4 @@
+let yearOfBirth = 1990;
+let yearFuture = 2025;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture )
```

Comment:
Check Template literals and how you can write these statements in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals